release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -511,7 +514,12 @@ The new registry, `registry.redhat.io`, requires authentication for access to
 images and hosted content on {product-title}. Following the move to the new
 registry, the existing registry will be available for a period of time.
 
-See Accessing and Configuring the Red Hat Registry for more information.
+See xref:../install_config/configuring_red_hat_registry.adoc#install-config-configuring-red-hat-registry[Authentication Enabled Red Hat Registry]
+for more information.
```

Comment:
i assume this is what i'm reviewing? looks fine.

Reply:
Yeah, thanks!